src/main/java/seedu/address/logic/parser/FilterArticleCommandParser.java

```diff
@@ -21,8 +21,11 @@ public FilterArticleCommand parse(String userInput) throws ParseException {
                 PREFIX_TAG, PREFIX_START, PREFIX_END);
         if (!arePrefixesPresent(argMultimap, PREFIX_STATUS, PREFIX_TAG, PREFIX_START, PREFIX_END)) {
             throw new ParseException("Invalid command format!\nfilter: Applies a filter. "
-                    + "Parameters: S/Status TAG/tag ST/Start date EN/End date"
-                    + "\nExample: filter -a S/DRAFT TAG/Product Releases ST/01-01-2001 EN/03-03-2023"
+                    + "Parameters:" + PREFIX_STATUS + "Status " + PREFIX_TAG + "tag "
+                    + PREFIX_START + "Start date " + PREFIX_END + "End date"
+                    + "\nExample: filter -a " + PREFIX_STATUS
+                    + "DRAFT " + PREFIX_TAG + "Product Releases "
+                    + PREFIX_START + "01-01-2001 " + PREFIX_END + "03-03-2023"
```

Comment:
Maybe change all the lowercase "Status " etc. to all caps to standardise with other command error messages.

Reply:
Done. Thanks for the catch